### DISCUSSION TITLE
Amiberry RTG "goes missing"

### DISCUSSION THREAD

**Discussion by StusBrainDump** [BlitterStudio/amiberry#1358] [Q&A] [answered]
I have an issue where the RTG graphics board setting just resets. I've created an a1200 config from quickstart, and added an RTG. Config saved, loaded again and all settings saved correctly and present. Start emulation, get the default Amiga insert disk screen. F12, check config and RTG has disappeared! (shows "-" under RTG). Quit emulation, load config, RTG is back! Run emulation, F12, RTG gone again.

This happens on macOS on an intel macbook pro and an M3 Max macbook pro.

Am I doing something wrong somewhere?

**Comment by midwan** [comment, marked as answer]
Do you have 24-bit addressing enabled, in the CPU panel?
RTG cannot work with that option enabled, as you might notice that you can select a card, but not assign any VRAM to it. On startup, these options are validated and fixed where necessary, which is why you see the difference.

Disable 24-bit addressing and try setting an RTG card again, with some VRAM assigned to it.

**Reply by StusBrainDump** [reply]
You are a wonderful human, thank you :) Was driving me crazy!
